components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/edit/EditToolbar.kt

```diff
@@ -59,6 +59,7 @@ class EditToolbar internal constructor(
     internal val rootView: View
 ) {
     private val logger = Logger("EditToolbar")
+    private var autoHideEditActionEnd = false
```

Comment:
I am wondering if this should just be a property to the constructor and then we don't need `setAutoHideEditActionEnd`. No strong opinions on this.

Reply:
My original thought was the same too.  But implementation gets complicated since in Fenix we don't directly create `EditToolbar`.  We will have to go through `BrowserToolbar` to do that which adds more complexity.